assets/js/translator.js

```diff
@@ -687,9 +1096,6 @@ function translateText(ignoreIfEmpty) {
                 }
             }, endpoint);
         }
-        else {
```

Comment:
Why did this go away?

Reply:
Sorry, probably something I removed for debugging at some point.